### PR TITLE
[BE] BD proposals role definition

### PR DESCRIPTION
**Pull request by hotzevzl**
Draft of BD proposals role - I based this on the spreadsheet with our review of responsibilities/etc from the FE team, but ended up proposing quite a few more details based on coversations we've had these past months.

### CODE REVIEW

**Review by agnessa — APPROVED** [review]
Now I feel like a complete slacker for only including 24 lines of text in the strategy role proposal, vs Andrea's 90 😅 Great description, maybe also add a line here: https://github.com/Vizzuality/devismos/blob/7e72e5f59a36cb9140c89859ab543186d1765061/_docs/backenders/strategy-2021/roles/index.md

**Comment by hotzevzl** [comment]
@agnessa I am sorry 😱 I was stuck in a flight, Docker let me down for "real work" and all I could do was focusing on this role definition 😬 

added draft page to index: 52fae5985e6626d357cd068e9cb1d97cd931570f

File: _docs/backenders/strategy-2021/roles/business-development-proposals.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+- Ensure that the backend group is collaborating actively and efficiently in
+  business development process (see "Bernat's Dekalog" document)
+
+- Ensure that the business development team is up to date on the backend
+  strategy so that they can pursue leads that match capabilities and
+  ongoing direction of the team
+
+- Review the BD financial dashboard
+
+  *This may be more relevant to the Capacity role, in terms of promised
```
Comment (aagm) [inline _docs/backenders/strategy-2021/roles/business-development-proposals.md:50]: I do agree that in this point capacity rol will need fluent info with the bd representative in order to be able to evaluate and be able to forecast the capacity.

**Review by aagm — APPROVED** [review]

**Review by ikas — APPROVED** [review]
LGTM :+1:

**Review by alexeh — APPROVED** [review]

**Review by tiagojsag — APPROVED** [review]

File: _docs/backenders/strategy-2021/roles/business-development-proposals.md (hunk 1 above)
Comment (hotzevzl) [reply _docs/backenders/strategy-2021/roles/business-development-proposals.md:50]: @aagm should we move this responsibility to the Capacity role then? Or we can leave this as is - it doesn't hurt to review the financial dashboard periodically in any case, and we can adjust/remove this later if not really relevant.

Comment (aagm) [reply _docs/backenders/strategy-2021/roles/business-development-proposals.md:50]: Leave as it is but noting the need for fluid information between both roles

**Pull request merged**